crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
 settings_tab_columns_title:
   en: "Columns"
   fr: "Colonnes"
   tr: "Sütunlar"
+  it: "Colonne"
 settings_tab_tui_desc:
   en: "Settings which control how data is displayed in this Tui"
   fr: "Paramètres qui contrôlent la façon dont les données sont affichées dans ce Tui"
   tr: "Arayüzde verilerin nasıl görüntülendiğini kontrol eden ayarlar"
+  it: "Impostazioni che controllano come i dati vengono visualizzati in questo Tui"
 settings_tab_trace_desc:
   en: "Settings which control the tracing strategy"
   fr: "Paramètres qui contrôlent la stratégie de traçage"
   tr: "İzleme stratejisini kontrol eden ayarlar"
+  it: "Impostazioni che controllano la strategia di tracciamento"
 settings_tab_dns_desc:
   en: "Settings which control how DNS lookups are performed"
   fr: "Paramètres qui contrôlent la façon dont les recherches DNS sont effectuées"
   tr: "DNS aramalarının nasıl yapıldığını kontrol eden ayarlar"
+  it: "Impostazioni che controllano come vengono eseguite le ricerche DNS"
 settings_tab_geoip_desc:
   en: "Settings relating to GeoIp"
   fr: "Paramètres relatifs à GeoIp"
   tr: "GeoIp ile ilgili ayarlar"
+  it: "Impostazioni relative a GeoIp"
 settings_tab_bindings_desc:
   en: "Tui key bindings"
   fr: "Raccourcis clavier Tui"
   tr: "Tui tuş ayarları"
+  it: "Collegamenti chiave Tui"
 settings_tab_theme_desc:
   en: "Tui theme colors"
   fr: "Couleurs du thème Tui"
   tr: "Tui tema renkleri"
+  it: "Colori del tema Tui"
 settings_tab_columns_desc:
   en: "Tui table columns.  Press [%{c}] to toggle a column on or off and use the [%{d}] and [%{u}] keys to change the column order."
   fr: "Colonnes de table Tui. Appuyez sur [%{c}] pour activer ou désactiver une colonne et utilisez les touches [%{d}] et [%{u}] pour changer l'ordre des colonnes."
   tr: "Tui tablo sütunları. Bir sütunu açmak veya kapatmak için [%{c}] tuşuna basın ve sütun sırasını değiştirmek için [%{d}] ve [%{u}] tuşlarını kullanın."
+  it: "Colonne della tabella Tui. Premi [%{c}] per attivare o disattivare una colonna e usa i tasti [%{d}] e [%{u}] per cambiare l'ordine delle colonne."
 settings_table_header_setting:
   en: "Setting"
   fr: "Paramètres"
   tr: "Ayar"
+  it: "Impostazione"
 settings_table_header_value:
   en: "Value"
   fr: "Valeur"
   tr: "Değer"
+  it: "Valore"
 column_host:
   en: "Host"
   fr: "Hôte"
   tr: "Ana bilgisayar"
+  it: "Host"
 column_loss_pct:
   en: "Loss%"
   fr: "% Perdus"
   tr: "Kayıp%"
+  it: "Perdita%"
 column_snd:
   en: "Snd"
   fr: "Envoyés"
   tr: "Gönderilen"
+  it: "Inviare"
 column_recv:
   en: "Recv"
   fr: "Reçus"
   tr: "Alınan"
+  it: "Ricevere"
 column_last:
   en: "Last"
   fr: "Dernier"
   tr: "Son"
+  it: "Ultimo"
 column_avg:
   en: "Avg"
   fr: "Moyenne"
   tr: "Ort"
+  it: "Media"
 column_best:
   en: "Best"
   fr: "Meilleur"
   tr: "En iyi"
+  it: "Migliore"
 column_wrst:
   en: "Wrst"
   fr: "Pire"
   tr: "En kötü"
+  it: "Peggiori"
 column_stdev:
   en: "StDev"
   fr: "ÉcTyp"
   tr: "StDev"
+  it: "StDev"
```

Comment:
In theory this would be translated as `Deviazione standard`, but I think it's better to leave it in english